src/components/Card/Card.tsx

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import style from './Card.module.css';
 
 interface CardProps {
+  customStyle?: React.CSSProperties | undefined;
```

Comment:
? をつけてる時点で optional (つまり undefined も含まれている) ので、
```suggestion
  customStyle?: React.CSSProperties;
```
で OK